### DISCUSSION TITLE
Chevereto reporting incorrect IP

### DISCUSSION THREAD

**Discussion by JasonHK** [chevereto/chevereto#63] [Q&A]
Chevereto giving incorrect IP even the `CHEVERETO_HEADER_CLIENT_IP` environment variable was set correctly (giving TWO IPs instead of ONE).

![image](https://github.com/chevereto/chevereto/assets/4410086/aef7119d-b131-42ac-9fae-c2aa03b43347)

**Comment by rodber** [comment]
The effect of `CHEVERETO_HEADER_CLIENT_IP` is that you instruct [where to look for the IP](https://github.com/chevereto/chevereto/blob/303d4322c1fffe52ac3211113addc127073faf70/app/src/Legacy/G/functions.php#L781). It doesn't format your value, it just takes what you populated on the header.

Basically, your server setups is wrong as you should be passing one IP for that header.
